CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) 
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
```

Comment:
Maybe we should order the changes

1. BC breaks
2. Security
3. New features
4. Everything else